vagrant/provisioning/roles/ldap/files/userXlsxToInstallerYaml.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python
+import sys
+import openpyxl
+
+if len(sys.argv) < 3:
+    print("Must provide Excel input file name and starting row")
+    sys.exit(1)
+
+excel_file = sys.argv[1]
+start_row = int(sys.argv[2])
+
+wb = openpyxl.load_workbook(excel_file)
+sheet = wb.active
+
+def fnFirstInitLnFirstX(firstName, lastName, numLnChars):
+    """Construct a samAccountName by first char of first name, up to seven chars 
+    of last name"""
+    return (firstName[0:1] + lastName[0:numLnChars]).lower()
+
+def getCellValue(sheet, row, column):
+    """Get value of a cell at the given index"""
+    return sheet[openpyxl.utils.cell.get_column_letter(column) + str(row)].value
+
+def formatGroupProperty(group):
+    """Format a group name with the proper ldap_prefix and ldap_group_base placeholders"""
+    return '      - "CN={{ ldap_prefix }}' + group[0:group.find('@')] + ',{{ldap_group_base}}"'
+
+for r in range(start_row, sheet.max_row + 1):
+    firstName = getCellValue(sheet, r, 1)
+    lastName = getCellValue(sheet, r, 2)
+    email = getCellValue(sheet, r, 3)
+    groups = getCellValue(sheet, r, 6)
+    office = getCellValue(sheet, r, 8)
```

Comment:
`office` is EEOC-specific ... when we support another customer this way we may end up making the script more generic.  Or maybe each customer will have a one-off based on this example.